cmd/kubeadm/app/cmd/reset.go

```diff
@@ -217,7 +217,10 @@ func (r *Reset) Run(out io.Writer, client clientset.Interface, cfg *kubeadmapi.I
 
 		If your cluster was setup to utilize IPVS, run ipvsadm --clear (or similar)
 		to reset your system's IPVS tables.
-
+		
+		The reset process does not clean your kubeconfig files. 
+		If you wish to reset the profile, you must remove it manually.
+		Please check the $HOME/.kube directory and remove it.
```

Comment:
Last line should read:
`Please, check the contents of $HOME/.kube/config file.`

Reply:
Thanks @rosti the expression is more appropriate!

Reply:
please remove the trailing space after `kubeconfig files.` on the first line.

i would change this to:
> The reset process does not clean your kubeconfig files and 
> you must remove them manually. Please, check the contents
> of $HOME/.kube/config file.

Reply:
@neolit123 I already fix it thanks!